client/nsxt_client.go

```diff
@@ -42,3 +43,18 @@ func (c *nsxtClient) GetDhcpStatus(dhcpID string, localVarOptionals map[string]i
 	dhcpServerStatus, _, err := c.apiClient.ServicesApi.GetDhcpStatus(c.apiClient.Context, dhcpID)
 	return dhcpServerStatus, err
 }
+
+func (c *nsxtClient) ReadClusterStatus() (administration.ClusterStatus, error) {
+	clusterStatus, _, err := c.apiClient.NsxComponentAdministrationApi.ReadClusterStatus(c.apiClient.Context, nil)
```

Comment:
for nil `localVarOptionals` are we going to hide it from the interface signature?

Reply:
let's use YAGNI principle for this https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it
we can throw things that don't matter for this (in this case: localVarOptionals) for ReadClusterStatus method

Reply:
ack, will update existing PR